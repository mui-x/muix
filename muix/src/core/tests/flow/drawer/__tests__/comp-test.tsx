/** @jsx createElement */

import { createElement } from 'reactxx-sheeter'

import { shallow } from '../../../../../tests/jestsetup'

import Comp from '../comp'

describe('Comp 1', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Comp />)
        expect(wrapper).toMatchSnapshot()
    })
})

test('Comp 2', () => {
    const wrapper = shallow(<Comp />)
    expect(wrapper).toMatchSnapshot()
})
