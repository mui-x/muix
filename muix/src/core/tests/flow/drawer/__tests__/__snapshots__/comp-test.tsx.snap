// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comp 1 renders correctly 1`] = `<div />`;

exports[`Comp 2 1`] = `<div />`;
